pkg/identity/keystone/sync.go

```diff
@@ -153,6 +156,13 @@ func (s *Syncer) syncData(u *userInfo) error {
 		}
 	}
 
+	for _, p := range s.syncConfig.ProjectNameBlackList {
+		if u.Extra["alpha.kubernetes.io/identity/project/name"][0] == p {
```

Comment:
Please use the const var `ProjectName`

Reply:
@lingxiankong I rebase master branch and use `ProjectName` instead of `alpha.kubernetes.io/identity/project/name`. You can review it if you have time. thanks.

Reply:
@lingxiankong which doc should i add or update? Can you tell me, thanks.

Reply:
Because the namespace synchronization feature is not mentioned in the current k8s-keystone-auth doc, what I suggest is you could add a new section after `Authorization policy definition(version 2)` named `Namespace synchronization`, add a brief description and a subsection named `Configuration` and describe the new config option you added in this PR